bower.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "kuzzle-backoffice",
+  "name": "kuzzle admin console",
   "description": "A handy administrative console for Kuzzle.io",
```

Comment:
Why keep Kuzzle.io here, if we change it later?

Reply:
good catch. ill change it.